src/analyses/libraryFunctions.ml

```diff
@@ -29,6 +29,19 @@ let c_descs_list: (string * LibraryDesc.t) list = LibraryDsl.[
     ("iswalnum", unknown [drop "wc" []]);
     ("iswprint", unknown [drop "wc" []]);
     ("rename" , unknown [drop "oldpath" [r]; drop "newpath" [r];]);
+    ("puts", unknown [drop "s" [r]]);
+    ("strspn", unknown [drop "s" [r]; drop "accept" [r]]);
+    ("strcspn", unknown [drop "s" [r]; drop "accept" [r]]);
+    ("strtod", unknown [drop "nptr" [r]; drop "endptr" [w_deep]]);
+    ("strtol", unknown [drop "nptr" [r]; drop "endptr" [w_deep]; drop "base" []]);
+    ("strtoll", unknown [drop "nptr" [r]; drop "endptr" [w_deep]; drop "base" []]);
+    ("strtoul", unknown [drop "nptr" [r]; drop "endptr" [w_deep]; drop "base" []]);
+    ("strtoull", unknown [drop "nptr" [r]; drop "endptr" [w_deep]; drop "base" []]);
```

Comment:
I think these functions only change which character `endptr` points to (`w`) but don't modify those pointed to characters, which would be a double dereference (`w_deep`). So `**endptr` should not be modified because it's a pointer into the constant `nptr` string.